@xen-orchestra/web-core/lib/locales/en.json

```
@@ -415,6 +415,7 @@
   "new-vm": "New VM",
   "new-vm.add": "Add a new VM",
   "new-vm.description": "VM description",
+  "new-vm.feature-not-supported": "This is our new VM creation form. Cloud-init configuration, Advanced options and vTPM management are not yet supported. Updated version will be available soon in 2026. In the meanwhile, full feature set is available here in",
```

To avoid a sentence without an end, you can use a [special component](https://vue-i18n.intlify.dev/guide/advanced/component) provided by VueI18n.
Here you can define a _slot name_ that will be reused in the template:

```suggestion
  "new-vm.feature-not-supported": "This is our new VM creation form. Cloud-init configuration, Advanced options and vTPM management are not yet supported. Updated version will be available soon in 2026. In the meanwhile, full feature set is available here in {xo-5}",
```